dcos/dcos-ui
# DCOS-49577: [1.13] MesosLogContainer / TaskFileViewer never loads logs for failed tasks

Add a check to prevent infinite loading
and a message for missing files.

Closes https://jira.mesosphere.com/browse/DCOS-49577

## Testing
1. Start a service.
2. Open the service and click on an instance.
3. In `plugins/services/src/js/pages/task-details/TaskFileViewer.js`, change line 150 from `const files = this.getLogFiles();` to `const files = []`, to mock missing files.
4. Open the logs tab for that instance.
5. Verify that it doesn't load forever and a message is shown.

## Trade-offs
None.

## Dependencies
None.

## Screenshots
### Before
![screenshot from 2019-03-05 14-15-26](https://user-images.githubusercontent.com/40791275/53804980-9d91f500-3f51-11e9-8beb-593e8918565f.png)

### After
![screenshot from 2019-03-05 14-09-48](https://user-images.githubusercontent.com/40791275/53805002-ad113e00-3f51-11e9-97a7-2d0bae464eab.png)
